types/architecture.pp

```diff
@@ -0,0 +1,9 @@
+type Poudriere::Architecture = Enum[
+  'amd64',
+  'arm',
+  'arm64',
```

Comment:
where is this list from?
are these only Tier 1 Architectures?

Reply:
I built this list by seeking what packages where available on the FreeBSD mirrors.  Probably not good, and I forgot that there is this `target.target_arch` notation that diverges from `$facts['os']['architecture']`.

I updated this list with what `make targets` output, added a snippet of code to copy-paste in order to rebuild this list if needed, and made the whole parameter optional to take all this into account!